MSEdgeDriver/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$osVersion = [System.Environment]::OSVersion.Version
+$major = $osVersion.Major
+
+if ($major -ne 10) {
+    throw 'Unsupported Windows version.'
+}
+
+# Binaries are x86, we need to install them in the right path depending on the OS architecture
+if ([Environment]::Is64BitProcess) {
+    $childpath = 'SysWOW64'
+    $url = 'https://az813057.vo.msecnd.net/webdriver/msedgedriver_x64/msedgedriver.exe'

```

Comment:
I have some concerns about this. Let me know what you think:
* We don't do any checksum to validate the download is correct --> We will have to update this package every time there's a new release
* We will always download this file no matter the current installed version of chrome --> If we update that binary and the dev is using an old version it will not work properly. 

Maybe we could do something a bit smarter where we check the installed version(s) and download the ones that are appropriate with fallbacks? Another alternative is to publish a new version of the package each time that there's a new release and have the `version` in the xml in sync so it will make it easier for people to download the right one.
I think we should have a meeting about the maintenance of this before merging and publishing. What do you think?